lib/survicate_sdk.dart

```diff
@@ -78,9 +80,9 @@ class SurvicateSdk {
   /// To change a trait, clients need to send the same key with a different value.
   /// See also:
   ///  * [setUserTraits], the method used to set multiple traits at once.
-  static void setUserTrait(String key, String value) async {
+ static void setUserTrait(UserTrait trait) async {
     try {
-      await SurvicateSdkPlatform.instance.setUserTrait(key, value);
+      await SurvicateSdkPlatform.instance.setUserTrait(trait.key, trait.value ?? "");
```

Comment:
It's not possible to set a null trait in Flutter?

Reply:
In Flutter itself it is but iOS expects NonNull String values